pkg/apiserver/proxy.go

```diff
@@ -56,6 +56,7 @@ func (r *ProxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	var apiResource string
 	var httpCode int
 	reqStart := time.Now()
+	pretty := isPrettyPrint(req)
```

Comment:
A little concerned about proxy looking at query parameters.  I would say proxy shouldn't do this (otherwise if he backend also supports pp it's going to get mangled).